app/(candidate)/dashboard/website/domain/components/DomainProvider.js

```diff
@@ -0,0 +1,20 @@
+'use client'
+
+import { createContext, useContext, useState } from 'react'
+
+export const DomainContext = createContext({
+  domain: null,
+  setDomain: () => {},
+})
+
+export const useDomain = () => useContext(DomainContext)
```

Comment:
Where is this being used? 

I'm not seeing it being called anywhere in this PR? 🤔 

I see `useDomainStatus`, but no calls to `useDomain`....

Reply:
If it's not called it would beg the question as to whether or not `DomainProvider` or `DomainContext` are needed, right?